WcaOnRails/app/views/new_registration_mailer/send_registration_mail.html.erb

```diff
@@ -0,0 +1,25 @@
+<p>
+  <%= t 'users.mailer.create_new_account.welcome', user_email: @user.email %>
+</p>
+
+<p>
+  <%= t 'users.mailer.create_new_account.confirm' %>
+</p>
+
+<p>
+  <%= link_to t('users.mailer.create_new_account.confirmation_link'), confirmation_url(@user, confirmation_token: @user.confirmation_token, locale: I18n.locale) %>
+</p>
+
+<p>
+  <%= t 'users.mailer.create_new_account.activate' %>
+
+  <%= link_to('login form', 'https://www.worldcubeassociation.org/users/sign_in') %>
```

Comment:
nit: This won't work for environments with a different url (such as staging or local development). It would be better to use `new_user_registration_url` here instead.